src/ocean/OceanCompute.ts

```diff
@@ -17,6 +17,19 @@ export enum ComputeJobStatus {
     Deleted
 }
 
+export interface Output {
+    publishAlgorithmLog?: boolean
+    publishOutput?: boolean
+    brizoAddress?: string
+    brizoUri?: string
+    metadata?: Record<string, any>
+    metadataUri?: string
+    nodeUri?: string
+    owner?: string
+    secretStoreUri?: string
+    whitelist?: Array<string>
```

Comment:
just for consistency, we always do this for array of strings:

```suggestion
    whitelist?: string[]
```